pkg/virtctl/imageupload/imageupload.go

```diff
@@ -709,7 +710,14 @@ func getAndValidateUploadPVC(client kubecli.KubevirtClient, namespace, name stri
 	if !createPVC {
 		_, err = client.CdiClient().CdiV1beta1().DataVolumes(namespace).Get(context.Background(), name, metav1.GetOptions{})
 		if err != nil {
+			// When the PVC exists but the DV doesn't, there are two possible outcomes:
 			if k8serrors.IsNotFound(err) {
+				// 1. The DV was already garbage-collected. The PVC was created and populated by CDI as expected.
+				dvGarbageCollected := pvc.Annotations[deleteAfterCompletionAnnotation]
+				if dvGarbageCollected == "true" {
```

Comment:
You can also do
```go
if pvc.Anntations[deleteAfterCompletionAnnotation] == "true" {
...
}
```

Reply:
Considering the previous comment I'll do something like:
```go
if dvGarbageCollected := pvc.Annotations[deleteAfterCompletionAnnotation] == "true" &&
    pvc.Annotations[PodPhaseAnnotation] == string(v1.PodSucceeded); dvGarbageCollected {
```
wdyt?